buildenv/jenkins/JenkinsfileBase

```diff
@@ -108,34 +108,63 @@ def setupEnv() {
 def setupParallelEnv() {
 	stage('setupParallelEnv') {
 		timestamps{
-			setupEnv()
+			setupEnv()		
+			int iterate = 1
+			if (params.ITERATIONS) {
+				iterate = "${params.ITERATIONS}".toInteger()
+			}
+			//Initialization for things in common between both multiple iterations and many subtests
+			//repeatAmount is the number of times the loops go through
+			int repeatAmount = 0
+			int one = 1
+			
 			def testSubDirs = []
 			def testSubDirSize = 0
 			
-			//For parallel suppose BUILD_LIST is one specific category
-			dir("$WORKSPACE/openjdk-tests/${env.BUILD_LIST}") {
+			dir("$WORKSPACE/openjdk-tests/${env.BUILD_LIST}") 
+			{
 				testSubDirs = sh(returnStdout: true, script: "ls -d */").trim().tokenize()
 				testSubDirSize = testSubDirs.size()
 			}
-			echo "testSubDirSize is ${testSubDirSize}, testSubDirs is ${testSubDirs}, running test in parallel mode"
+			
+			//Setup for option-specific parameters
+			if (iterate == 1){
+				//For parallel suppose BUILD_LIST is one specific category
+				repeatAmount = testSubDirSize
+			}
+			else
+			{
+				repeatAmount = iterate
+			}
+			
+			//echo "testSubDirSize is ${testSubDirSize}, testSubDirs is ${testSubDirs}, running test in parallel mode"
```

Comment:
I think we should keep this echo.

Reply:
This is only useful in the situation where the iteration count is 1